src/template-tags/tickets.php

```diff
@@ -960,7 +961,7 @@ function tribe_tickets_get_readable_amount( $number, $mode = 'own', $display = f
 
 			$html[] = esc_html( $handler->unlimited_term );
 		} else {
-			$html[] = esc_html( $number );
+			$html[] = number_format_i18n( $number );
```

Comment:
Does this account for the commerce specific settings used by WC / TC on the ticket in question? That's the only problem I have here with resetting some of that (in case it differs from the site settings)

Reply:
`tribe_tickets_get_readable_amount()` is only used in ET or ET+ as an output, not to get a value and then run it through commerce-specific output. If a commerce provider wants its number to go through its own formatting function, it should use that instead.